packages/markdown-docx/src/ToCiceroMarkVisitor.js

```diff
@@ -180,6 +191,8 @@ class ToCiceroMarkVisitor {
     fetchFormattingProperties(node, nodeInformation) {
         for (const runTimeNodes of node.elements) {
             if (runTimeNodes.name === 'w:rPr') {
+                let colorCodePresent = false,
+                    highlightCodePresent = false;
```

Comment:
Declare them separately if the formatting tool is forcing them to be on separate lines.